tools/codegen/gen_backend_stubs.py

```diff
@@ -256,7 +263,44 @@ def gen_dispatcher_registrations(
 
     assert class_name is not None
     backend_index = backend_indices[dispatch_key]
+
+    dispatch_registrations_body = list(concatMap(
+        dest.RegisterDispatchKey(
+            backend_index,
+            Target.REGISTRATION,
+            selector,
+            rocm=False,
+            cpp_namespace=cpp_namespace,
+            class_method_name=f'{class_name}',
+            skip_dispatcher_op_registration=False),
+        grouped_native_functions
+    ))
+    deferred_dispatch_registrations = ""
+    static_init_dispatch_registrations = ""
+    if eager_registration:
+        static_template = CodeTemplate("""\
+TORCH_LIBRARY_IMPL(aten, $dispatch_key, m) {
+    $dispatch_registrations_body
+};""")
+        static_init_dispatch_registrations = static_template.substitute(
+            dispatch_key=dispatch_key,
+            dispatch_registrations_body=dispatch_registrations_body
+        )
+    else:
+        deferred_template = CodeTemplate("""\
+TORCH_API void Register${backend_name}${dispatch_key}NativeFunctions() {
+    auto m = MAKE_TORCH_LIBRARY_IMPL(aten, $dispatch_key);
+    $dispatch_registrations_body
```

Comment:
@wconstab I think this library variable needs to be global
```suggestion
static auto m = MAKE_TORCH_LIBRARY_IMPL(aten, $dispatch_key);
TORCH_API void Register${backend_name}${dispatch_key}NativeFunctions() {
    $dispatch_registrations_body
```
I made that change locally and it appeared to work for me